apis/placement/v1beta1/binding_types.go

```diff
@@ -42,6 +46,21 @@ type ResourceBindingSpec struct {
 	TargetCluster string `json:"targetCluster"`
 }
 
+// BindingState is the state of the binding
+type BindingState string
+
+const (
+	// BindingStateCreating means the binding is ready but need to be rolled
+	// out to the target cluster.
+	BindingStateCreating BindingState = "Creating"
```

Comment:
I assume this state is set by the scheduler. When will the scheduler change the state from Creating to Active?

Reply:
Scheduler doesn't. The rollout changes this to "active" when it is ready to roll it out.

Reply:
Is active the same as bound?

Basically creating == scheduled, active == bound?